middleware.ts

```diff
@@ -28,6 +29,14 @@ export async function middleware(req: NextRequest) {
     data: { session },
   } = await supabase.auth.getSession();
 
+  if (session?.user && req.nextUrl.pathname === "/account-deleted") {
+    // Delete the account from Supabase and log the user out.
+    await supabase.auth.admin.deleteUser(session.user.id);
```

Comment:
Just wanted to check if this is cool to run @brandonroberts @jpmcb or is there a soft delete we want here as well?